vendor/vendor.json

```diff
@@ -69,5 +69,5 @@
 			"revisionTime": "2017-05-12T22:20:15Z"
 		}
 	],
-	"rootPath": "github.com/shomali11/slacker"
+	"rootPath": "github.com/evandroflores/slacker"
```

Comment:
Just making sure that this is an intentional change. I thought the `rootPath` had to match the original repo's url